.github/workflows/test_publish_github_release.yml

```diff
@@ -52,4 +52,13 @@ jobs:
       sha: ${{ inputs.sha }}
       version_number: ${{ inputs.version_number }}
       changelog_path: ${{ inputs.changelog_path }}
-      testing: ${{ inputs.dry_run }}
+      test_run: ${{ inputs.dry_run }}
```

Comment:
Let's keep this consistent and call it `test_run` everywhere.

Reply:
I have renamed `dry_run` to `test_run` in https://github.com/dbt-labs/dbt-core-release-test/commit/22d5c2199f703144848bdc05725960d31568edee